rviz_default_plugins/src/rviz_default_plugins/displays/wrench/wrench_display.cpp

```diff
@@ -96,6 +100,7 @@ void WrenchDisplay::reset()
 
 void WrenchDisplay::updateWrenchVisuals()
 {
+  // bool accept_NaN = accept_NaN_values_->getBool();
```

Comment:
why is this commented out?

Reply:
This is kept as part of testing, will remove.